shared/cfg/guest-os/Linux/Fedora/30.cfg

```diff
@@ -0,0 +1,49 @@
+- 30:
+    variants:
+        - x86_64:
+            vm_arch_name = x86_64
+        - aarch64:
+            vm_arch_name = aarch64
+    image_name = images/f30-${vm_arch_name}
+    os_variant = fedora30
+    # default boot path is set in ../Fedora.cfg: boot_path = "images/pxeboot"
+    no unattended_install..floppy_ks
```

Comment:
this should have been commented out, right?

Reply:
Does it work? IIRC q35 does not support it at all and even on i440fx I'm not sure floppy works with F30...

Reply:
no idea, you're probably right, I just noticed the syntax is wrong